smartling-strings-api/src/test/java/com/smartling/api/strings/v2/StringsApiTest.java

```diff
@@ -0,0 +1,209 @@
+package com.smartling.api.strings.v2;
+
+import com.smartling.api.strings.v2.pto.AsyncStatusResponsePTO;
+import com.smartling.api.strings.v2.pto.CreatedStringsListPTO;
+import com.smartling.api.strings.v2.pto.GetSourceStringsCommandPTO;
+import com.smartling.api.strings.v2.pto.SourceStringListPTO;
+import com.smartling.api.strings.v2.pto.StringsCreationPTO;
+import com.smartling.api.strings.v2.pto.TranslationsCommandPTO;
+import com.smartling.api.strings.v2.pto.TranslationsPTO;
+import com.smartling.api.v2.client.ClientConfiguration;
+import com.smartling.api.v2.client.DefaultClientConfiguration;
+import com.smartling.api.v2.client.auth.BearerAuthStaticTokenFilter;
+import com.smartling.api.v2.response.ListResponse;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.apache.http.HttpStatus;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.ws.rs.core.HttpHeaders;
+
+import static org.junit.Assert.*;
+
+public class StringsApiTest
```

Comment:
You check only method and path from the request but you never check that the payload is serialized as expected
You also don't check how the response is deserialized

This API implementation is copypasted from the legacy project which is working and well tested
tests will be very helpful when we will switch major versions of Resteasy or Jackson

Reply:
well... it actually tests json marshalling (from public doc examples) and it will fail if for instance boolean property in DTO turns into Integer. In this case I just do not check that DTO contains exact payload from json that I provide. In this case examining DTO it turns into test for test